examples/platforms/nrf52811/openthread-core-nrf52811-config.h

```diff
@@ -181,23 +181,23 @@
 #endif
 
 /**
- * @def OPENTHREAD_CONFIG_HEAP_SIZE
+ * @def OPENTHREAD_CONFIG_INTERNAL_HEAP_SIZE
  *
  * The size of heap buffer when DTLS is enabled.
  *
  */
-#ifndef OPENTHREAD_CONFIG_HEAP_SIZE
-#define OPENTHREAD_CONFIG_HEAP_SIZE (4096 * sizeof(void *))
+#ifndef OPENTHREAD_CONFIG_INTERNAL_HEAP_SIZE
+#define OPENTHREAD_CONFIG_INTERNAL_HEAP_SIZE (4096 * sizeof(void *))
```

Comment:
nit: can we rename to `OPENTHREAD_CONFIG_HEAP_INTERNAL_SIZE` so that we have a common `OPENTHREAD_CONFIG_HEAP_*` prefix for all heap-related configs?

Reply:
Sure. Would it be better for you to drop the `INTERNAL` entirely or place it after `HEAP`?